portfolio/src/main/webapp/style.css

```diff
@@ -219,21 +219,38 @@ hr{
   display:none;
 }
 #comment_container{
-    background-color: #fdcb9e;
     border-left: 1px solid #3f3f44;
     border-right:1px solid #3f3f44;
     max-width: 80vw;
     margin:auto;
     margin-top:30px;
     margin-bottom:30px;
 }
-.comment{
+.posComment{
+  border-top: 1px solid #3f3f44;
+  border-bottom: 1px solid #3f3f44;
+  background-color: #87f165;
+  padding-bottom:10px;
+  max-width: 80vw;
+  margin:auto;
+}
+.neuComment{
+  border-top: 1px solid #3f3f44;
+  border-bottom: 1px solid #3f3f44;
+  background-color: #f1cf65;
+  padding-bottom:10px;
+  max-width: 80vw;
+  margin:auto;
+}
+.negComment{
     border-top: 1px solid #3f3f44;
     border-bottom: 1px solid #3f3f44;
+    background-color: #f18965;
     padding-bottom:10px;
     max-width: 80vw;
     margin:auto;
 }
+
```

Comment:
Considering these classes have very similar CSS, it would make sense to assign two classes to the comments, the first is the comment class and the second is the posComment, negComment, or neuComment class. (https://stackoverflow.com/questions/8722163/how-to-assign-multiple-classes-to-an-html-container)

Doing this will allow you to set CSS for the "comment" class that is common to all comments, then have very small CSS defined for each sub class that only changes the things that need to be specifically changed.